lib/wiki_edits.rb

```diff
@@ -113,8 +113,11 @@ def get_tokens(current_user)
     return { status: 'no current user' } unless current_user
     @access_token = oauth_access_token(current_user)
     get_token = @access_token.get("#{@api_url}?action=query&meta=tokens&format=json")
-
```

Comment:
I think the better approach here, to make this as specific as possible, will be something like returning or skipping the JSON parse if the response code is 503.

This method should always return a hash that includes the status (see the return line further down).

This is a good case for writing a failing test case first, and then proceeding with the solution.